src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
Consider renaming the enum to 'Color' instead of 'Colors' to follow Java naming conventions for enums. Enums are usually named in singular form. 🤓